experimental/LieAlgebras/src/AbstractLieAlgebra.jl

```diff
@@ -267,23 +318,74 @@ end
     lie_algebra(R::Field, dynkin::Tuple{Char,Int}; cached::Bool) -> AbstractLieAlgebra{elem_type(R)}
 
 Construct the simple Lie algebra over the ring `R` with Dynkin type given by `dynkin`.
```

Comment:
Docstring doesn't match changed signature
```suggestion
    lie_algebra(R::Field, S::Symbol, n::Int; cached::Bool) -> AbstractLieAlgebra{elem_type(R)}

Construct the simple Lie algebra over the ring `R` with Dynkin type given by `S` and `n`.
```